### ISSUE TITLE
The Assignment

### ISSUE THREAD

**Issue by Lauwy222** [Alset-Innovations/Plan-of-Approach#2]

**Comment by Lauwy222** [comment]
Writting this for now:
```LATEX
\section{The Assignment}
In the ambitious realm of the \textbf{PROJE4} course, \textbf{Alset Innovations} is tasked with the exciting challenge of designing and implementing a cutting-edge solution for Smart Energy Devices. The focal point of our project revolves around creating a sensored brushless-DC (BLDC) motor controller for an electric bicycle. This endeavor aligns seamlessly with the broader objective of utilizing intelligent embedded technology to enhance efficiency and conserve energy in various applications.

\subsection{Project Scope:}
Our primary goal is to develop a motor controller capable of seamlessly driving a \SI{24}{\volt}, \SI{78}{\watt} BLDC motor in a closed-loop configuration using hall sensor feedback. Our innovative twist lies in the selection of the \textbf{STM32F103C8T6} microcontroller, a new and unexplored avenue, which distinguishes us from our peers who may opt for the more conventional \textbf{RP2040}. This strategic decision places \textbf{Alset Innovations} at the forefront of innovation of this course.

% \subsection{Key Technical Requirements:}
% \begin{enumerate}
%     \item Utilize discrete transistors to control the three motor phases.
%     \item Design a custom printed circuit board (PCB) for the entire system, showcasing our prowess in hardware development.
%     \item Measure and display the current delivered to the motor through the user interface.
%     \item Implement a user-friendly control input (throttle) on the user interface for speed control.
%     \item Integrate the \textbf{STM32F103C8T6} microcontroller to process hall sensor signals, demonstrating our proficiency in microcontroller programming.
%     \item Deliver a comprehensive technical report detailing our design choices, simulations, schematics, and testing results.
% \end{enumerate}

\subsection{Strategic Approach:}
\textbf{Alset Innovations} recognizes the uniqueness of our project by opting for the \textbf{STM32} microcontroller, thereby charting a pioneering course in the implementation of smart energy devices. Our approach involves meticulous planning and execution, encompassing the following key phases:

\begin{enumerate}
    \item \textbf{Initial Research and Exploration:}
        \begin{itemize}
            \item Dive into the specifics of BLDC motor control, \textbf{STM32} microcontroller capabilities, and related electronic circuit theory.
            \item Explore the feasibility of our chosen microcontroller and its potential advantages in comparison to the \textbf{RP2040}.
        \end{itemize}

    \item \textbf{Hardware and PCB Design:}
        \begin{itemize}
            \item Research, compare and test different \textbf{MOSFETS} and \textbf{GATE} drivers.
            \item Develop a detailed schematic and layout for the PCB, focusing on optimal integration and functionality.
            \item Address the challenges posed by components like the \textbf{QFN-56} package of the \textbf{STM32} microcontroller.
        \end{itemize}

    \item \textbf{Software Development:}
        \begin{itemize}
            \item Craft robust code for the microcontroller, ensuring seamless motor control and user interface interaction.
            \item Conduct thorough testing on real prototype hardware, substantiating our software's reliability.
        \end{itemize}

    \item \textbf{Documentation and Reporting:}
        \begin{itemize}
            \item Maintain an exhaustive record of our design process, simulations, and results.
            \item Adhere to the specified template for the final technical report, showcasing our commitment to clarity and precision.
        \end{itemize}
\end{enumerate}

In summary, \textbf{Alset Innovations} stands poised to redefine the landscape of smart energy devices through our unique blend of innovation, strategic decision-making, and meticulous execution. Our Plan of Approach sets the stage for a transformative journey in the world of electrical engineering.
```

**Comment by Lauwy222** [comment]
Not sure if I should put this in: What is @LuukvanKappel and @justin040304 opinion?

```LATEX
% \subsection{Key Technical Requirements:}
% \begin{enumerate}
%     \item Utilize discrete transistors to control the three motor phases.
%     \item Design a custom printed circuit board (PCB) for the entire system, showcasing our prowess in hardware development.
%     \item Measure and display the current delivered to the motor through the user interface.
%     \item Implement a user-friendly control input (throttle) on the user interface for speed control.
%     \item Integrate the \textbf{STM32F103C8T6} microcontroller to process hall sensor signals, demonstrating our proficiency in microcontroller programming.
%     \item Deliver a comprehensive technical report detailing our design choices, simulations, schematics, and testing results.
% \end{enumerate}```
